src/sentry/static/sentry/app/components/charts/utils.jsx

```diff
@@ -0,0 +1,12 @@
+const DEFAULT_TRUNCATE_LENGTH = 80;
+
+export function truncationFormatter(value, truncate) {
+  if (!truncate) {
+    return value;
+  }
+  let truncationLength =
+    truncate && typeof truncate === 'number' ? truncate : DEFAULT_TRUNCATE_LENGTH;
+  return value.length > truncationLength
+    ? value.substring(0, truncationLength) + '…'
```

Comment:
Wonder if we should use the html entity `&hellip;` here for the ellipsis.

Reply:
I couldn't get it to work properly, I think it's a problem with echarts. I enabled their `useHTML` option to true, but it didn't work. Because it depends on that, I think this is safer.

Reply:
I looked into it and Echarts auto escapes its strings, so it doesn't look too doable unfortunately.

Reply:
yeah that's fine